indico/modules/search/client/js/components/results/Event.jsx

```diff
@@ -30,28 +32,26 @@ const renderDates = (startDt, endDt) =>
     </List.Item>
   );
 
-const Event = ({typeFormat: type, title, url, categoryPath, startDt, endDt, chairPersons}) => {
+const Event = ({typeFormat: type, eventId, title, categoryPath, startDt, endDt, chairPersons}) => {
   return (
     <div styleName="item">
       <List.Header styleName="header">
-        <a href={url}>{title}</a>
+        <a href={eventURL({event_id: eventId})}>{title}</a>
       </List.Header>
       <List.Description styleName="description">
-        {/* if it's a lecture print the list of speakers */}
         {type === 'lecture' && chairPersons.length !== 0 && (
           <List.Item>
-            {chairPersons.map(i => (
-              <div key={i.name}>
-                {i.title
-                  ? `${i.title} ${i.name} (${i.affiliation})`
-                  : `${i.name} (${i.affiliation})`}
+            {chairPersons.map(person => (
```

Comment:
We need the same deduplication logic like in contributions for name+affiliation here.